From 56ee05d917537dfe54bfc9e0803a2796408a6c81 Mon Sep 17 00:00:00 2001
From: Mike Looijmans <mike.looijmans@topic.nl>
Date: Tue, 26 May 2020 08:40:17 +0200
Subject: [PATCH] mach-zynqmp/spl.c: Remove 1-second delay at boot

SPL boot is slow on the zynqMP because of this delay. The initialization done
in psu_init no longer requires an external delay, so this can be removed to
speed up the SPL boot flow considerably.

Signed-off-by: Mike Looijmans <mike.looijmans@topic.nl>
---
 arch/arm/mach-zynqmp/spl.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/arm/mach-zynqmp/spl.c b/arch/arm/mach-zynqmp/spl.c
index 1d6d959b7d..061234783e 100644
--- a/arch/arm/mach-zynqmp/spl.c
+++ b/arch/arm/mach-zynqmp/spl.c
@@ -24,8 +24,6 @@ void board_init_f(ulong dummy)
 	debug_uart_init();
 	puts("Debug uart enabled\n"); /* or printch() */
 #endif
-	/* Delay is required for clocks to be propagated */
-	udelay(1000000);
 
 	/* Clear the BSS */
 	memset(__bss_start, 0, __bss_end - __bss_start);
-- 
2.17.1

